apps/src/code-studio/assets/SpriteManagementDirectory.jsx

```diff
@@ -1,7 +1,26 @@
 import React from 'react';
+import Button from '@cdo/apps/templates/Button';
+import color from '@cdo/apps/util/color';
+import {moveDefaultSpriteMetadataToProduction} from '@cdo/apps/assetManagement/animationLibraryApi';
 
 export default class SpriteManagementDirectory extends React.Component {
+  state = {
+    updated: false
+  };
+
+  moveChangesToProduction = () => {
+    //Files to move to production folder: defaultSprites.json
+    moveDefaultSpriteMetadataToProduction()
+      .then(() => {
+        this.setState({updated: true});
+      })
+      .catch(err => {
+        console.log(err);
```

Comment:
we might want to talk to kylie about the functionality we want here -- it seems confusing to have an error state give no feedback, especially when the action is a "scary" action (uploading to prod). i think the simplest thing we could do here is replace `updated` with an enum that indicates 3 states: nothing (no icon), success (checkmark icon), or error (X icon). we may also want to add a success/error message and loading state to the button, but i think that can be kylie's call